### PR TITLE
[MINOR][SQL] Improve the comments about null tracking for UnsafeRow

### PR DESCRIPTION
**Pull request by beliefer**
### What changes were proposed in this pull request?
This PR expect to improve the comments about null tracking for `UnsafeRow`.
The old comment of `UnsafeRow` have confused text `[null bit set]`.
In fact, the portion is a lot of bit or bit array which does't always be null.
On the other hand, it tell users nothing. we need the information more clear.

### Why are the changes needed?
Improve the comments about null tracking for `UnsafeRow`.

### Does this PR introduce _any_ user-facing change?
'No'.
Just update comments.

### How was this patch tested?
N/A

### CODE REVIEW

**Comment by beliefer** [comment]
ping @MaxGekk @gengliangwang @dongjoon-hyun cc @cloud-fan

**Comment by cloud-fan** [comment]
what's wrong with "null bit set"? A bit set for null tracking.

**Review by HyukjinKwon — COMMENTED** [review]
Yeah I don't particularly find this confusing too

File: sql/catalyst/src/main/java/org/apache/spark/sql/catalyst/expressions/UnsafeRow.java [hunk 1]
```diff
@@ -46,10 +46,10 @@
 /**
  * An Unsafe implementation of Row which is backed by raw memory instead of Java objects.
  *
- * Each tuple has three parts: [null bit set] [values] [variable length portion]
+ * Each tuple has three parts: [null-tracking portion] [values] [variable length portion]
```
Comment (cloud-fan) [inline sql/catalyst/src/main/java/org/apache/spark/sql/catalyst/expressions/UnsafeRow.java:49]:
```suggestion
 * Each tuple has three parts: [null-tracking bit set] [values] [variable length portion]
```

**Review by dongjoon-hyun — COMMENTED** [review]
I'm also not sure if this is an improvement or not, but I feel new sentence looks like becoming repeating itself a little.
```
The null-tracking portion is used for null tracking 
```

**Review by dongjoon-hyun — APPROVED** [review]
Since it's not a big deal, let's merge this for @beliefer and move on to more important PRs. :)

File: sql/catalyst/src/main/java/org/apache/spark/sql/catalyst/expressions/UnsafeRow.java [hunk 2]
```diff
@@ -46,10 +46,10 @@
 /**
  * An Unsafe implementation of Row which is backed by raw memory instead of Java objects.
  *
- * Each tuple has three parts: [null bit set] [values] [variable length portion]
+ * Each tuple has three parts: [null-tracking bit set] [values] [variable length portion]
  *
- * The bit set is used for null tracking and is aligned to 8-byte word boundaries.  It stores
- * one bit per field.
+ * The null-tracking bit set is used for null tracking and is aligned to 8-byte word boundaries.
```
Comment (cloud-fan) [inline sql/catalyst/src/main/java/org/apache/spark/sql/catalyst/expressions/UnsafeRow.java:51]:
```suggestion
 * The null-tracking bit set is aligned to 8-byte word boundaries.
```

Comment (cloud-fan) [reply sql/catalyst/src/main/java/org/apache/spark/sql/catalyst/expressions/UnsafeRow.java:51]: This is not addressed

Comment (cloud-fan) [reply sql/catalyst/src/main/java/org/apache/spark/sql/catalyst/expressions/UnsafeRow.java:51]: Again, this is not addressed

**Comment by cloud-fan** [comment]
thank, merging to master!

**Comment by beliefer** [comment]
@cloud-fan @dongjoon-hyun @HyukjinKwon Thank you !

**Pull request closed without merging**